### DISCUSSION TITLE
Single engine taxi out - SETWA

### DISCUSSION THREAD

**Discussion by tush0701** [flybywiresim/aircraft#4956] [General]
Hi,

Airbus has a functionality called "SETWA", which stands for Single Engine Taxi Without APU.

What is the procedure for it, and is it implemented in the A32NX, or shall we raise an enhancement request for it?

The A32NX is one of the greatest piece of work in the history of freeware, and kudos to the team!

**Comment by SuperSixBravo** [comment]
I am not dev on this project but I dont think it is implemented yet, Without the APU I think you need the Ground Starter not sure. But watch this:
https://forums.x-plane.org/index.php?/forums/topic/165846-video-single-engine-taxi-and-ptu-operation/

BTW the pilot is a real world pilot, and the SIM is Xplane, with the Toliss addon. It seems quite old 2 years ago, but its all been updated since then.
